CNA scope conflict improvement (#6612)
jenkins-infra/jenkins.io: content/security/for-maintainers.adoc

.. The security team provides a private repository for that work in the `jenkinsci-cert` GitHub organization.
.. Work usually happens on a branch, and a corresponding pull request will be used for review.
. A *date and time of the release is coordinated* between the security team and maintainers.

>   The security team handles CVE ID assignment (in cases where there is no CNA scope conflict), advance notification of users, and creation of the security advisory.

Author: > We still handled it by coordinating with the other CNA, we just didn't assign it ourselves.

Do you prefer an approach like this one ,or do you think it should not be mentioned?

```suggestion
  The security team handles CVE ID assignment (assignment may require coordination with another CNA in case of a scope conflict), advance notification of users, and creation of the security advisory.
```
